src/h5web/visualizations/MappedHeatmapVis.tsx

```diff
@@ -24,13 +24,22 @@ function MappedHeatmapVis(props: Props): ReactElement {
     keepAspectRatio,
     showGrid,
     resetDomains,
+    autoScale,
+    disableAutoScale,
   } = useHeatmapConfig();
 
   const baseArray = useBaseArray(dataset, value);
   const dataArray = useMappedArray(baseArray, mapperState);
-  const dataDomain = useDataDomain(dataArray.data as number[]);
+  const dataDomain = useDataDomain(
+    (autoScale ? dataArray.data : baseArray.data) as number[]
+  );
```

Comment:
I feel we might do something better here to use the fact that `dataDomain` needs not to be recomputed if `autoScale` is `false`

Reply:
What do you not like about this? Not sure I understand.

Reply:
Nevermind, I had the false impression that `dataDomain` was recomputed as each slice change when `autoScale` is false.

It is fine !